src/index.tsx

```diff
@@ -127,19 +159,28 @@ function PrefabProvider({
           onError(reason);
         });
     }
-  }, [apiKey, loadedContextKey, contextKey, loading, setLoading, onError]);
+  }, [
+    apiKey,
+    loadedContextKey,
+    contextKey,
+    loading,
+    setLoading,
+    onError,
+    prefabClient?.instanceHash,
+  ]);
 
   const value: ProvidedContext = React.useMemo(
     () => ({
-      isEnabled: prefab.isEnabled.bind(prefab),
+      isEnabled: prefabClient.isEnabled.bind(prefabClient),
       contextAttributes,
-      get: prefab.get.bind(prefab),
-      getDuration: prefab.getDuration.bind(prefab),
-      keys: Object.keys(prefab.configs),
-      prefab,
+      get: prefabClient.get.bind(prefabClient),
+      getDuration: prefabClient.getDuration.bind(prefabClient),
+      keys: Object.keys(prefabClient.configs),
+      prefab: prefabClient,
       loading,
+      settings,
     }),
-    [loadedContextKey, loading, prefab]
+    [loadedContextKey, loading, prefabClient, settings]
```

Comment:
I'm not great at Memos, but should this be dependent on `prefabClient?.instanceHash` instead of `prefabClient` like above?